changelog/march2024/2024-03-22-kubernetes-changed-routed-ips-now-default-on-new-kubern.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Routed IPs now default on new Kubernetes clusters
```

Comment:
```suggestion
title: Routed IPs are available by default on Kubernetes clusters
```

Reply:
I think it's more than just available, they are the default IP type you will get (if I have understood correctly) so maybe "Routed IPs are now default on Kubernetes clusters" or potentially "provisioned by default"